operating/manage-user-access/manage-role-based-access-control-3062.adoc

```diff
@@ -1,13 +1,20 @@
 [id="manage-role-based-access-control"]
-= Managing role-based access control
+= Managing role-based access control in {product-title} 3.0.62 and older
 include::modules/common-attributes.adoc[]
 :context: manage-role-based-access-control
 
 toc::[]
 
 [role="_abstract"]
-{product-title} comes with role-based access control (RBAC) that you can use to configure roles and grant various levels of access to {product-title} to different users.
-These roles govern access to {product-title} resources to prevent unwanted access.
+{product-title} (RHACS) comes with role-based access control (RBAC) that you can use to configure roles and grant various levels of access to {product-title} to different users.
```

Comment:
Maybe "to different users" --> "for different users" so you aren't repeating "to" again? It was sort of hard to read this the first time.